automated/linux/piTest/test-PT-1.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+source "../test_format.sh"
+
```

Comment:
For this test we should add ready to use configuration files, so that pictory is not needed. Those files can live in a git repository or even in the test-repository. A test will use this file, copy it to the correct place and restart the driver. We can't assume that a device is every time configured, especially not after an image update.

Reply:
I have added the configuration details for core3-rack01.